Prox/Prox/AppDelegate.swift

```diff
@@ -61,7 +49,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationDidEnterBackground(_ application: UIApplication) {
         // if there is a timer running, cancel it. We'll wait until background app refresh fires instead
         locationMonitor.cancelTimeAtLocationTimer()
```

Comment:
I think this "time at location" was so that we would fetch events if the user stayed at a place long enough. I think we can remove it.